pkg/ext-proc/backend/inferencemodel_reconciler.go

```diff
@@ -0,0 +1,55 @@
+package backend
+
+import (
+	"context"
+
+	"inference.networking.x-k8s.io/llm-instance-gateway/api/v1alpha1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/client-go/tools/record"
+	"k8s.io/klog/v2"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+type InferenceModelReconciler struct {
+	client.Client
+	Scheme         *runtime.Scheme
+	Record         record.EventRecorder
+	Datastore      *K8sDatastore
+	ServerPoolName string
+	Namespace      string
+}
+
+func (c *InferenceModelReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Result, error) {
+	if req.Namespace != c.Namespace {
+		return ctrl.Result{}, nil
+	}
+	klog.V(1).Info("reconciling InferenceModel", req.NamespacedName)
+
+	service := &v1alpha1.InferenceModel{}
+	if err := c.Get(ctx, req.NamespacedName, service); err != nil {
+		klog.Error(err, "unable to get InferencePool")
```

Comment:
pls also log the req.NamespacedName. 

Even better, I believe the controller-runtime sets up a contextual logger already, which automatically logs the request attributes. You should be able to get it with 

```
import	"sigs.k8s.io/controller-runtime/pkg/log"

logger := log.FromContext(ctx)
```

Reply:
We use `klog` throughout the EPP, I don't think we should deviate here. If we want to change this, we could just use `klog`s implementation, it has contextual logging.

Feel free to cut an issue as this is unrelated to API changes, it's in the diff simply due to the naming shift.